content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/deploy-corda-cluster.md

```diff
@@ -0,0 +1,388 @@
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
+        external-dns.alpha.kubernetes.io/hostname: corda.example.com
+```
+
+### PostgreSQL
+
+The password for PostgreSQL can be specified directly as a Helm override but this is not recommended. Instead, create a Kubernetes secret containing the password with a key of password. By default, the install expects a database called `cordacluster` but this can be overridden. You can then define the PostgreSQL configuration as follows:
+```yaml
+db:
+  cluster:
+    host: <POSTGRESQL_HOST>
+    port: 5432
+    user: <POSTGRESQL_USER>
+    database: <POSTGRESQL_CLUSTER_DB>
+    existingSecret: <POSTGRESQL_PASSWORD_SECRET_NAME>
+```
+
+### Kafka
+
+Specify the Kafka bootstrap servers as a comma-separated list:
+```yaml
+kafka:
+  boostrapServers: <KAFKA_BOOTSTRAP_SERVERS>
+```
+If desired, a prefix can be applied to all of the Kafka topics used by the Corda deployment. This enables multiple Corda clusters to share a Kafka cluster. For example:
+```yaml
+kafka:
+  topicPrefix: <KAFKA_TOPIC_PREFIX>
+```
+Enable TLS if required by the Kafka client protocol:
+```yaml
+kafka:
+  tls:
+    enabled: true
+```
+
+If the broker certificate is self-signed or can not be trusted for some other reason, create a Kubernetes secret containing the client trust store. The trust store can be in PEM or JKS format. If JKS format is used, you can supply a password for the trust store. The following example is for a trust store in PEM format stored against the `ca.crt` key in the Kubernetes secret:
+```yaml
+kafka
+  tls:
+    secretRef:
+      name: <TRUST-STORE-SECRET-NAME>
+      key: "ca.crt"
+    type: PEM   
+```
+
+Corda supports SASL for Kafka authentication. If your Kafka instance requires SASL authentication, create a secret containing the credentials with the username and password of the key and then specify the secret name, along with the required mechanism in the overrides:
+
+```yaml
+kafka:
+  sasl:
+    enabled: true
+    username:
+      valueFrom:
+        secretKeyRef:
+          name: <SASL-SECRET-NAME>
+          key: "username"
+    password:
+      valueFrom:
+        secretKeyRef:
+          name: <SASL-SECRET-NAME>
+          key: "password"
+    mechanism: "SCRAM-SHA-512"
+```
+
+### Bootstrapping
+
+By default, the Helm chart automatically configures Kafka, PostgreSQL, and a default set of Corda RBAC roles as part of the deployment process.
+If desired, each of these steps can be disabled and the necessary [configuration performed manually](manual.html).
+
+#### Kafka
+The Kafka bootstrapping creates the topics required by Corda.
+If `kafka.topicPrefix` has been specified, the process uses this as a prefix for all of the topic names.
+The bootstrap configuration enables the default number of topic partitions to be overridden.
+You may need to increase this to support a larger number of Corda worker replicas.
+It is also possible to override the default topic replica count.
```

Comment:
```suggestion
It is also possible to override the default topic replica count
```
Line 223 is not a full sentence so I'd join it to the previous sentence using a semi-colon. ("...replica count; for example, if....")